tests/test_recorders.py

```diff
@@ -162,11 +164,15 @@ def test_numpy_recorder(simple_linear_model):
     model.run()
 
     assert rec.data.shape == (365, 1)
+    assert cost_rec.data.shape == (365, 1)
     assert np.all((rec.data - 10.0) < 1e-12)
+    np.testing.assert_allclose(cost_rec.data, np.ones_like(cost_rec.data) * -2.0)
 
     df = rec.to_dataframe()
+    df2 = cost_rec.to_dataframe()
     assert df.shape == (365, 1)
     assert np.all((df.values - 10.0) < 1e-12)
+    np.testing.assert_allclose(df2.values, np.ones_like(cost_rec.data) * -2.0)
```

Comment:
Is it worth having a test where the node cost varies?

Reply:
Done.